README.adoc

```diff
@@ -64,7 +64,14 @@ This will set up
 
 ==== 2. Download monitoring-engine
 
-Download `remoting-opentelemetry-engine-<tag>.jar` from link:https://github.com/jenkinsci/remoting-opentelemetry-plugin/releases[Release Page].
+Download `remoting-opentelemetry-engine.jar` from Jenkins maven repository.
+
+:version: 1.0-rc66.0edcb5079f9e
```

Comment:
Is there a simple way to download the "latest" version from the maven repository?